src/main/java/uk/gov/hmcts/probate/validator/CaveatEmailValidationRule.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.probate.validator;
+import uk.gov.hmcts.probate.model.ccd.caveat.request.CaveatDetails;
+
+public interface CaveatEmailValidationRule {
+    String REGEX = "[a-z0-9!#$%&'*+/=?^_`{|}~-]{1,30}(?:\\.[^.\\n]{1,30}){0,30}@[a-z0-9](?:[a-z0-9-.]{0,30}[a-z0-9])?\\.[a-z0-9](?:[a-z0-9-]{0,10}[a-z0-9])?";
```

Comment:
are these 2 regex actually the same? if we would try to apply the same validation for both caveat and grant cases, then best to use 1 definition, not 2